R/toolReadSetFromGDX.R

```diff
@@ -1,36 +1,60 @@
 #' toolReadSetFromGDX
 #'
-#' For the given fulldata gdx file, this function returns a dataframe with the
-#' values_parameters of some variables from fulldata.
+#' For the given gdx file, this function returns a dataframe with the
+#' values_parameters of some variables from the gdx file.
+#'
+#' @param gdx_file Name of the gdx file
+#'
+#' @param set sets to read from gdx file
 #'
 #' @return Dataframe with the values_parameters of some variables
-#' from fulldata.
+#' from the gdx file.
 #'
 #' @author Anastasis Giannousakis Fotis Sioutas
 #'
 #' @examples
 #' \dontrun{
-#' a <- toolReadSetFromGDX()
+#' a <- toolReadSetFromGDX(gdx_file = "fulldata", set = c("SECTTECH", "INDSE"))
 #' }
 #'
 #' @importFrom gdx readGDX
 #'
 #' @export
 
-toolReadSetFromGDX <- function() {
+toolReadSetFromGDX <- function(gdx_file = "fulldata", set = "SECTTECH") {
+
+  a <- readGDX(paste0(gdx_file, ".gdx"))
```

Comment:
suggested change: reading the whole GDX file is very slow, better read only the items that the user wants. Also, remove `paste0` from this line so that users can provide either a filename or a full path as `gdx_file`